plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -11,7 +11,7 @@ const {
 const config = {
 	timeout: DEFAULT_TIMEOUT_OVERRIDE
 		? Number( DEFAULT_TIMEOUT_OVERRIDE )
-		: 90 * 1000,
+		: 300 * 1000,
```

Comment:
Just curious, why do we need to increase the timeout value here?

Reply:
The ever loveable create-variable-product.spec.js was timing out without an increase.  Frustrating, but this is a *maximum* timeout, so it doesn't affect overall runtime really.

Reply:
I also realized that since this PR intends to run the tests through the UI, then it makes sense to increase the max timeout. I can confirm that one of the tests inside `create-variable-product.spec.js` (specifically, the `can manually add a variation, manage stock levels, set variation defaults and remove a variation` test) sometimes almost reaches the 90sec limit even on a passing run. So it does make sense to increase the timeout in UI mode.

I guess we could improve the way we increase this timeout by specifying `DEFAULT_TIMEOUT_OVERRIDE` in the `test:e2e-pw-ui` script so that the increase would only happen when the tests are run in UI mode. If we'll agree with this approach, we can do it in a small, separate PR instead. This PR contains some PNPM lock changes that might get more merge conflicts soon. I'm ok with increasing the timeout within the Playwright config file for now.